metro.config.js

```diff
@@ -79,7 +79,7 @@ module.exports = {
     getTransformOptions: async () => ({
       transform: {
         experimentalImportSupport: false,
-        inlineRequires: false,
+        inlineRequires: true,
```

Comment:
C'est pour quoi ça ?

Reply:
c'est préisé dans la doc, j'avoue que je n'ai pas creusé